docs/index.rst

```diff
@@ -106,23 +106,79 @@ You can use pip and make to quickly set up a development environment::
 
 Please adhere to the following guidelines:
 
-* Lint your code with flake8 and pylint. This can be done easily by executing
-  ``make lint``. The former linter must pass with zero warnings, and warnings
-  from the latter linter should be minimized.
-* Check for errors by running the unit test suite. This can be done easily by
-  executing ``make test``. It must pass with zero errors.
-* Document your code. Documentation can be easily generated by executing ``make
-  docs-html``.
-* Unit tests are *very* highly recommended.
+* All PR’s should follow the predetermined pull request template and explain the problem that is addressed. Issues should follow template and explain what the problem is.
+* Maintain Coding Standards
+    * Keep pep8 rules
+    * Follow the same stylistic and logical patterns used in the code
+        * All entity class names and class attributes have to be in the singular format
+        * All required entity attributes have to have `required=True` parameter
+        * It is preferable to use `alpha` data type for default string values for easier debug procedure
+        * In case any workaround is introduced, it is necessary to provide corresponding BZ ID directly into the code docstring
+        * All linting and formatting/style checks would be enforced by Travis-CI and PR would be considered broken until checks are passed successfully.
 * Adhere to typical commit guidelines:
-
-    * Commits should not cause NailGun to break.
+    * Commits should not cause NailGun’s unit test to fail. If it does, it will the responsibility of contributor to review those failures and fix them in the same PR's or raise another. The tracking of failures would be responsibility of contributor.
     * Commits should be small and coherent. One commit should address one issue.
     * Commits should have `good commit messages`_.
     * `Rebasing`_ is encouraged. Rebasing produces a much nicer commit history
       than merging.
+* To make the review process easy for all reviewers and anyone else interested in the new functionality, please provide some output making use of your changes. Having example of usage in docstring along with your code could really him other to build up onyour code. You can add log from Python interactive shell or some tests results (from Robottelo / Foreman Ansible Modules) in PR message, or you can do something completely different - as long as it runs your code, it's fine!
```

Comment:
Couple of typos got in second sentence. Should be:

> Having example of usage in docstring along with your code could really help others to build up on your code.

Reply:
I am always in a hurry, thanks :)